Fix broken import in "common" package

### Current Behavior

Running `pnpm install` fails on the prepare step due to a broken import

### Expected Behavior

A clean `pnpm install` runs without issue.

### To Reproduce

Run `pnpm install` at top level.
My console reports the following error:
```
│ . prepare:bundle: packages/common bundle: ✘ [ERROR] Could not read from file: /<<repo-location>>/packages/common/src/Registry.ts
│ . prepare:bundle: packages/common bundle:     src/index.ts:3:14:
│ . prepare:bundle: packages/common bundle:       3 │ export * from './Registry';
```
### Context

_No response_

--- a/packages/common/src/index.ts
+++ b/packages/common/src/index.ts
@@ -1,3 +1,3 @@
 // import './test';
 
-export * from './Registry';
+export * from './registry';

--- a/packages/common/src/test.ts
+++ b/packages/common/src/test.ts
@@ -1,5 +1,5 @@
 import { Expression } from '@configu/sdk';
-import { Registry } from './Registry';
+import { Registry } from './registry';
 
 const test = async () => {
   await Registry.register('./module.js');

